ip_ranges.go

```diff
@@ -0,0 +1,45 @@
+package netaddr
```

Comment:
To be consistent, this file should probably be be name `iprange.go`. It might not have been the best, but we should be consistent.